README.md

```diff
@@ -98,6 +98,65 @@ Here `ClangCompiler` can be replaced with one of `"Compiler" /. CCompilers[Full]
 
 A less frequently updated version is available through the Wolfram's public paclet server and can be installed by running `PacletInstall["SetReplace"]`.
 
+# Symbols and Functions
+
+## SetReplace, SetReplaceList, SetReplaceAll, SetReplaceFixedPoint, SetReplaceFixedPointList
+
+`SetReplace` (and related `SetReplaceList`, `SetReplaceAll`, `SetReplaceFixedPoint` and `SetReplaceFixedPointList`) are the functions the package is named after. They are quite simple, don't have a lot of options, and simply perform replacement operations either one-at-a-time (as in the case of `SetReplace`), to all non-overlapping subsets (`SetReplaceAll`), or until no more matches can be made (`SetReplaceFixedPoint`). A suffix `*List` implies the function will return a set after each replacement instead of just the final result.
+
+These functions are good for their simplicity, but we don't use them much anymore as a more advanced `WolframModel` incorporates all of these features plus other utilities helpful for the exploration of our models.
```

Comment:
I for some reason don't like this sentence. I think it is a little negative..  I would probably say something like this. 

These functions are good for their simplicity and can be primarily used for its low level usage. WolframModel is advanced version of SetReplace and incorporates ... ... of our models.

Reply:
Good point. No need to criticize the first introduced function. I've updated the sentence similar to your suggestion.